src/xcode/ENA/ENA/Source/Services/Risk/Provider/__tests__/RiskProviderTests.swift

```diff
@@ -133,39 +99,42 @@ final class RiskProviderTests: XCTestCase {
 
 		let exposureSummaryProvider = ExposureSummaryProviderMock()
 
-		let expectThatNoSummaryIsRequested = expectation(description: "expectThatNoSummaryIsRequested")
-		expectThatNoSummaryIsRequested.isInverted = true
+		let detectionRequested = expectation(description: "expectThatNoSummaryIsRequested")
```

Comment:
Please explain this. Why do you change the variable name but not the string value for the description? Is this the same? Why add this mental mapping?